src/commandProcessExecution.ts

```diff
@@ -52,7 +52,7 @@ export class CommandProcessExecution {
     }
   }
 
-  async complete(): Promise<string> {
+  async complete(stdin?: string): Promise<string> {
```

Comment:
Maybe more future proof would be to pass this to the `constructor` instead. That way we can also use it in `spawn` method if necessary.

Reply:
Sounds good. I pushed a commit for that!